Sources/GDBRemote/Session.cpp

```diff
@@ -1541,6 +1542,32 @@ void Session::Handle_qCRC(ProtocolInterpreter::Handler const &,
 }
 
 //
+// Packet:        qFileLoadAddress:<file_path>
+// Description:   Returns the load address of a memory mapped file.
+// Compatibility: LLDB
+//
+void Session::Handle_qFileLoadAddress(ProtocolInterpreter::Handler const &,
+                                      std::string const &args) {
+  if (args.empty()) {
+    sendError(kErrorInvalidArgument);
+    return;
+  }
+
+  Address address;
+  ErrorCode error =
+      _delegate->onQueryFileLoadAddress(*this, HexToString(args), address);
+  if (error == kSuccess && !address.valid()) {
```

Comment:
Is this useful? Is it possible to have `onQueryFileLoadAddress` return `kSuccess` and for the `address` to be invalid?